app/javascript/components/ResultsPage.tsx

```diff
@@ -111,9 +111,9 @@ class ResultsPage extends React.Component<ResultsPageProps, Nothing> {
         </GridContainer>
 
         <div id='results'>
-          {this.props.results.map(result => {
+          {this.props.results.map((result, index) => {
             return (
-              <div className='result'>
+              <div className='result' key={index}>
```

Comment:
This key was added to fix a React warning for unique keys when iterating over an array.